transcom/mymove
# B 18429 SSW Model to Service Object Conversion - INT

## [B-18429 SSW Model to Service Object Conversion](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A883383)

## Summary

Is there anything you would like reviewers to give additional scrutiny? - If any data that I have manually translated from PersonallyProcuredMove to PPMShipment was incorrectly converted, and if any errors are caused in testing. The old SSW Generation was not functional, the restructuring is more important than functionality for this ticket, as the next downstream also covers SSW functionality for filling the form. Please run the SSW testing suite. Also, I commented some deprecated SSW Form Filling tests, but did not delete them since the next downstream will require their recreation. If this is incorrect, please let me know and I will remove them entirely.

The SSW was converted from a model to a service object, and as much data as possible was changed from personallyprocuredmove to PPMShipment. There are still obligations and signed certifications that will require downstream work, but this restructure was more time sensitive due to downstream dependencies.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Pull the branch, ensure the SSW files are correctly structured and no personallyprocuredmove code is still active. 
2. Run the SSW Testing suite, ensure all tests pass and all tests have been reviewed such that data is account for excluding obligations and signed certifications.

### Frontend - Should not apply for this PR.


### Backend - Please alert me to any erroneous comments! 


### Database

#### Any new migrations/schema changes:


## Screenshots
